source/_posts/Vehicle-Parking-Occupancy-Detection.md

```diff
@@ -11,7 +11,7 @@ tags:
 ---
```

Comment:
Link to   previous content - in  content there should be link with previous content
-



-

Reply:
Done